### PR TITLE
Devfile Ecosystem update

### PR DESCRIPTION
**Pull request by kim-tsao**
Signed-off-by: Kim Tsao <ktsao@redhat.com>

**What does this PR do / why we need it**:

Introduces a new Devfile Ecosystem section to explain the big picture

**Which issue(s) this PR fixes**:

Fixes #?

https://github.com/devfile/api/issues/819

**PR acceptance criteria**:

- [ ] Unit Tests
- [ ] E2E Tests
- [ ] Documentation

**How to test changes / Special notes to the reviewer**:

Tested locally with with `yarn dev` and `yarn start`

### CODE REVIEW

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+'''''
+The following is an example of a devfile ecosystem that is framed within the context of an organization looking to standardize their development environment.  In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If the organization sees the need to author and share their own devfiles internally, they will need a role based approach where Developers, Devfile Authors, and Registry Administrators can interact together.
+
+image::devfileEcosystem.svg[]
+
+
+=== *The Main Areas of the Devfile Ecosystem*
+=== *Create*
+
+A devfile author aka “stack provider” can be an individual or a group representing a runtime vendor.  This role requires sound knowledge of the supported runtime so devfiles can be created with the proper guidance on how to build and run applications.
```
Comment (jc-berger) [inline webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:12]: `aka` is too informal. Simply say, `A devfile author or "stack provider"...` or `A devfile author, also known as "stack provider," can be...` 

For the second sentence, I'd rephrase so that the focus is more on the role. IBM SG wants us to focus on the user whenever we can. What do you think of this rewrite?: 

`Devfile authors need sound knowledge of the supported runtime so they can create devfiles to build and run applications.`

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+If a runtime stack is not available in the public registry, an organization can choose to develop their own and keep it private for their in-house development.
+
+=== *Share*
+
+The public community registry is managed by the community and hosted by Red Hat.  This is one way to share your devfile if there is a benefit to the greater community.
```
Comment (jc-berger) [inline webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:18]: For the second sentence, I'd put the action on the user. I think that'd motivate them more to work with devfiles. What do you think of this rephrase?:

`Share your devfile to the public community registry so other teams can benefit from your application.`

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+If an organization wants to keep their own devfiles private but wishes to share with other departments, they can assign a Devfile Registry Administrator to deploy, maintain, and manage the contents of their private registry and the default list of devfile registries available in a given cluster.  With this set up, they can use devfiles from both the public and private registries, which gives them the flexibility of options.
```
Comment (jc-berger) [inline webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:20]: There's nothing wrong with the first sentence, so definitely feel free to keep it as is. However, it is a little on the long side. So you could divide it into smaller sentences, something like this: `If an organization wants to keep their own devfiles private but wishes to share with other departments, they can assign a Devfile Registry Administrator. The Devfile Registry Administrator deploys, maintains, and manages the contents of their private registry and the default list of devfile registries available in a given cluster.`

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+=== *Consume*
+
+Devfiles can be made available to developers via supported tools*^*^*.  Many of the existing tools offer a way to register or catalog public/private Devfile Registries which then allows the tool to expose the devfiles for development.
```
Comment (jc-berger) [inline webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:24]: Put the attention on the user/role, avoid latin phrases like "via, per, etc.", and I'm not sure what `*^*^*' means? 

`Developers use the supported tools to access devfiles.` 

For the second sentence, the IBM SG wants us to avoid `/` so simply say, `...a way to register or catalog public and private devfile registries...`

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+The following is an example of a devfile ecosystem that is framed within the context of an organization looking to standardize their development environment.  In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If the organization sees the need to author and share their own devfiles internally, they will need a role based approach where Developers, Devfile Authors, and Registry Administrators can interact together.
```
[the remark this answers is not in the document]
Comment (kim-tsao) [reply webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc]: Please review the updated text.  I simplified the first line to `Organizations looking to standardize their development environment can do so by adopting devfiles.` and incorporated all other changes.

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc (hunk 1 above)
Comment (kim-tsao) [reply webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc]: I decided to use the term `runtime provider` so we're consistent with the persona title.

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc (hunk 2 above)
Comment (kim-tsao) [reply webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc]: done

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc (hunk 3 above)
Comment (kim-tsao) [reply webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc]: Updated with some modifications.  De-capitalized the title and removed `Devfile` just to keep it consistent with the existing title

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc (hunk 4 above)
Comment (kim-tsao) [reply webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc]: I was trying to  bold and superscript the asterisk :smile: .   I updated it to remove the bold effect since it wasn't rendering.  It should look like this: 
image
with an explanation at the bottom of the section: 
image

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc [hunk 6]
```diff
@@ -0,0 +1,34 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+Organizations looking to standardize their development environment can do so by adopting devfiles.
+In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If your organization needs custom devfiles that are authored and shared internally, then you will need a role based approach so developers, devfile authors, and registry administrators can interact together.
```
Comment (jc-berger) [inline webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:5]: I think we can avoid future tense since we've already specified a condition (if your organization needs custom devfiles). So, 
`If your organization needs custom devfiles that are authored and shared internally, then you need a role based approach...`

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+image::devfileEcosystem.svg[]
+
+
+=== *The Main Areas of the Devfile Ecosystem*
+=== *Create*
+
+A devfile author, also known as a runtime provider can be an individual or a group representing a runtime vendor.  Devfile authors need sound knowledge of the supported runtime so they can create devfiles to build and run applications.
```
Comment (jc-berger) [inline webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:13]: Missing comma:

`A devfile, author, also known as a runtime provider, can be an individual...`

**Review by jc-berger — DISMISSED** [review]
/lgtm

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc (hunk 6 above)
Comment (kim-tsao) [reply webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:5]: updated

File: webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc (hunk 7 above)
Comment (kim-tsao) [reply webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc:13]: updated.  Thanks again for the very thorough review @jc-berger !

**Review by schultzp2020 — APPROVED** [review]
